client/jetpack-cloud/sections/agency-dashboard/sites-overview/index.tsx

```diff
@@ -21,10 +29,15 @@ import './style.scss';
 export default function SitesOverview(): ReactElement {
 	const translate = useTranslate();
 	const dispatch = useDispatch();
+	const isMobile = useMobileBreakpoint();
 	const jetpackSiteDisconnected = useSelector( checkIfJetpackSiteGotDisconnected );
 	const isPartnerOAuthTokenLoaded = useSelector( getIsPartnerOAuthTokenLoaded );
 	const purchasedLicense = useSelector( getPurchasedLicense );
 
+	const highlightFavoriteTab = getQueryArg( window.location.href, 'highlight' ) === 'favorite-tab';
+
+	const [ hightLightTab, setHightLightTab ] = useState( false );
```

Comment:
Is there a typo here?

Should it be `highlightTab` and `setHighlightTab`?

Reply:
Were you able to find where this logic is being used? I removed it and didn't find any regression (both desktop and mobile).

This code was removed in: 09ad234a5ec3a6048bbe2b7471b077ea831c70ac

I'll tag @yashwin to take a look when he's back, but I think it might be some WIP work.

Reply:
Yes, I didn't find any regression either. Looks good now. Thanks

Reply:
Sorry, just noticed a regression after you removed that logic. 

It no longer shows all the sites, it only shows Favorite sites on mobile view.

Reply:
Thanks! The commit was reverted and the typo was fixed in a8d2dbe4dd7feacdd78e61d61878f03f45e9d9e1